libs/youtube_utils.py

```diff
@@ -17,7 +17,8 @@ def get_youtube_entry(youtubeid, url='https://www.googleapis.com/youtube/v3/vide
     logger = logging.getLogger('social_website')
     if youtubeid != "":
         try:
-            response = urllib2.urlopen(''.join([url, youtubeid, '&key=', GOOGLE_API_KEY]))
+            response = urllib.urlopen(''.join([url, youtubeid, '&key=', GOOGLE_API_KEY]))
+            # print(response.read())
```

Comment:
Can we remove this commented print statement, as i could understand it was for debugging?

Reply:
Yes sir,we can remove the print statement as it was only for debugging purpose.